src/backport.ts

```diff
@@ -344,19 +344,30 @@ export class Backport {
             this.config.pwd,
           );
           if (pushExitCode != 0) {
-            const message = this.composeMessageForGitPushFailure(
-              target,
-              pushExitCode,
-            );
-            console.error(message);
-            successByTarget.set(target, false);
-            await this.github.createComment({
-              owner: workflowOwner,
-              repo: workflowRepo,
-              issue_number: pull_number,
-              body: message,
-            });
-            continue;
+            try {
+              // If the branch already exists, ignore the error and keep going.
+              await this.git.fetch(
+                branchname,
+                this.config.pwd,
+                1,
+                this.getRemote(),
+              );
```

Comment:
💭 When this continues, it does not consider the case where the remote branch differs from the local branch. This introduces a risk to creating a PR for a branch that does not contain the contents created by the backport-action. However, it requires write access to the repo to push such a branch to the remote. We can thus consider it not dangerous. A potential future improvement would be verifying the branch's contents.